libraries/AP_HAL_SITL/SITL_State.h

```diff
@@ -195,6 +195,8 @@ class HALSITL::SITL_State {
     VectorN<readings_wind,wind_buffer_length> buffer_wind_2;
     uint32_t time_delta_wind;
     uint32_t delayed_time_wind;
+    uint8_t wind_start_delay;
```

Comment:
maybe explicitly initiate this to 0

Reply:
I was told many many times that there is no need to init anything to zero because it is done automatically :)
But I don't mind at all

Reply:
On vehicule code yes as it prevent to waste space, but on SITL we aren't limited. And on those case, I prefer to do it explicitly as all the logic is based on this ! ^^ My opinion ... I don't know what the other would say...

Reply:
I went further

Reply:
@khancyr I get your point, but we don't do it anywhere, so it's clearly not required, and, to be honest, it keeps the code cleaner.